### ISSUE TITLE
java.lang.NullPointerException when trying to use built-in text editor

### ISSUE THREAD

**Issue by rugk** [nextcloud/android#5554]
I've tried to open a file in the text editor, I.e. .txt file.
However that file had a red cross as it was downloaded before and AFAIK modified locally.
So that may have caused some internal conflicts.

Reproducible.

************ CAUSE OF ERROR ************
```java
java.lang.NullPointerException: Attempt to invoke virtual method 'void androidx.appcompat.widget.SearchView.setOnQueryTextListener(androidx.appcompat.widget.SearchView$OnQueryTextListener)' on a null object reference
	at com.owncloud.android.ui.preview.PreviewTextFileFragment$TextLoadAsyncTask.onPostExecute(PreviewTextFileFragment.java:216)
	at com.owncloud.android.ui.preview.PreviewTextFileFragment$TextLoadAsyncTask.onPostExecute(PreviewTextFileFragment.java:149)
	at android.os.AsyncTask.finish(AsyncTask.java:695)
	at android.os.AsyncTask.access$600(AsyncTask.java:180)
	at android.os.AsyncTask$InternalHandler.handleMessage(AsyncTask.java:712)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:193)
	at android.app.ActivityThread.main(ActivityThread.java:6746)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```
************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30100190
Build flavor: generic

************ DEVICE INFORMATION ************
Brand: Fairphone
Device: FP3
Model: FP3
Id: 8901.2.A.0110.20200109
Product: FP3

************ FIRMWARE ************
SDK: 28
Release: 9
Incremental: 01092024

**Comment by tobiasKaminsky** [comment]
Does this happen when your retry it?
Judging from code it should make no difference if the file has a sync conflict.
I fear it is just a single runtime problem…

**Comment by rugk** [comment]
Yes, it is always reproducible.

Also I just saw when trying again there is a popup shown. Likely the one for the sync conflict. For one second, before it crashes.

When I opened a different txt file that was _not_ downloaded already and (thus) has no sync conflict, the built-in editor works.

**Comment by tobiasKaminsky** [comment]
Steps to reproduce
- have a text file with sync conflict
- click on it

@rugk thanks for reporting!

**Comment by joshtrichards** [comment]
Fixed in #5822 (or thereabouts).
